src/Progress/Store.js

```diff
@@ -1,36 +1,76 @@
-import _ from 'lodash/fp'
 import { combineReducers } from 'redux'
 import BaseStore from '../Core/BaseStore'
 
 export const identifier = 'progress'
+
 const baseStore = BaseStore(identifier)
 
-const isVisible = d => d.code < 200 || d.code >= 400
-const getTotal = _.sumBy( () => 1 )
-const getDone = _.sumBy( e => e.erfuellt )
-const dashboardData = _.flow([
-    _.over([ getTotal, getDone ]),
-    ([ total, done ]) => ({ total, done })
-])
+function moduleIsVisible(module) { return module.code < 200 || module.code >= 400; }
+
+function getTotal(data) { return data.length; }
+
+function getDone(data) {
+
+    let count = 0;
+
+    for (let i = 0; i < data.length; i++)
```

Comment:
besser: 
data.forEach() // falls data eine Map ist
for (progressDatum in data) { console.log(progressDatum) } //falls data ein array ist